styles/style.css

```diff
@@ -0,0 +1,246 @@
+* {
+    margin:0;
+    padding: 0;
+}
+
+.centered {
```

Comment:
Consider calling this something like `.centered-container`. `.centered` on its own sounds like something that we would want to apply to the content we want to center rather than the container it sits in.